INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+## How to validate App is running
+``
+     http://localhost:30007
```

Comment:
It would be helpful to specify what users should expect when accessing this URL to validate that the app is running. For example, mention if there is a specific page or response they should see.